client/src/App.js

```diff
@@ -112,7 +112,13 @@ class App extends Component {
     return (
       <div className="App">
         <main>
-          <Route path="/" exact component={SplashPage}></Route>
+          <Route
+          path="/"
+          exact
+          render={(routerProps) => (
+            <SplashPage loggedIn={this.state.logged_in} {...routerProps}/>
+          )}
+          ></Route>
```

Comment:
Could we reverse the order here? Better to redirect people to the splash page if they are not logged in than what we're doing here. We did it this way originally so that we'd have something up while we were building the app. But it's more likely that you're going to be visiting this page and be logged in.